articles/quickstart/native/ios-swift/01-login.md

```diff
@@ -15,7 +15,7 @@ contentType: tutorial
 useCase: quickstart
 ---
 
-<!-- markdownlint-disable MD041 -->
+<!-- markdownlint-disable MD002 MD041 -->
```

Comment:
```suggestion
<!-- markdownlint-disable MD041 -->
```

MD002 has been deprecated and is disabled by default. See https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md002---first-heading-should-be-a-top-level-heading

Reply:
This repo [uses 0.15.0](https://github.com/auth0/docs/blob/master/yarn.lock#L2562) - which throws an error from me when the git commit hook runs, because it's not deprecated in that version https://github.com/DavidAnson/markdownlint/blob/v0.15.0/doc/Rules.md#md002---first-heading-should-be-a-top-level-heading